client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -88,6 +89,26 @@ export class CheckoutThankYouHeader extends PureComponent {
 			);
 		}
 
+		if ( this.isBulkDomainTransfer() ) {
+			return (
+				<>
+					<span>
+						{ sprintf(
+							translate(
+								"We got it from here! We'll let you know when your newly transferred %s ready to use."
+							),
+							purchases?.length > 1 ? 'domains are' : 'domain is'
```

Comment:
This definitely doesn't work in Arabic. Should use `_n`.

Reply:
Should be working now 👍